pkg/porter/run.go

```diff
@@ -113,6 +113,9 @@ func (p *Porter) Run(opts RunOptions) error {
 		if err != nil {
 			return errors.Wrap(err, "unable to resolve sourced values")
 		}
+		// Hand over values needing masking in context output streams
+		p.Context.SetSensitiveValues(p.Manifest.GetSensitiveValues())
```

Comment:
Ok, so this is in reference to the comment above about refactoring. What do you think about having `p.Context.AppendSensitiveValue("poop")`? Then as the manifest finds sensitive words, instead of the manifest tracking them in a slice, and having to coordinate passing it back to the context here (and then also after resolving the dependency's values), it could append to them as it finds them?

You are closer to this, so let me know if you think that would help reduce complexity or not.

Reply:
In talking about this offline, I believe we came to the consensus that we're good with revisiting this area of the code after some near-term refactoring.